libs/Microsoft.MixedReality.WebRTC.Native/src/media/external_video_track_source.cpp

```diff
@@ -141,6 +139,10 @@ ExternalVideoTrackSourceImpl::~ExternalVideoTrackSourceImpl() {
   StopCapture();
   GlobalFactory::Instance()->RemoveObject(ObjectType::kExternalVideoTrackSource,
                                           this);
+}
+
+void ExternalVideoTrackSourceImpl::FinishCreation() {

```

Comment:
Do we need the extra concept of finishing creation? Will it do anything other than call the existing StartCapture?

Reply:
We do not want to expose the `StartCapture()` concept because all video track sources start in the capturing state in WebRTC, so I think we shouldn't return to the user a paused/stopped track source to avoid them creating a track from it. So we hide that under the "FinishCreation" concept.